### PR TITLE
#13 - Render environment variables in template locations

### PR DESCRIPTION
**Pull request by vassilevsky**
For #13 

Not sure if I need to whitelist variables. Let me know.

### CODE REVIEW

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#20](https://github.com/g4s8/gitstrap/issues/20) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) everybody who has role `REV` is banned at [#20](https://github.com/g4s8/gitstrap/issues/20); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: gitstrap.go [hunk 1]
```diff
@@ -72,6 +75,22 @@ func (y *Config) ParseFile(name string) error {
...
 }
 
+// RenderEnvVars - replace environment variable references in config with their values
+func (y *Config) RenderEnvVars() {
+	for i, template := range y.Gitstrap.Templates {
+		y.Gitstrap.Templates[i].Location = renderEnvVars(template.Location)
+	}
+}
+
+func renderEnvVars(str string) string {
```
Comment (g4s8) [inline gitstrap.go]: @vassilevsky we need a unit test to verify that it's working

**Comment by vassilevsky** [comment]
Which document describes this process? Thank you for the comment, but it's just a comment, and where is the guide? I have read through the entire policy and it says nothing about it 😕

**Comment by g4s8** [comment]
@vassilevsky check https://www.zerocracy.com/2019/05/22/code-review-tips.html

> Demand Unit Tests
>
>Each new feature or a bug fix should be started with a unit test. If a unit test is not present, the reviewer needs to fork the PR branch and ensure that the code works. But a good reviewer is lazy and doesn’t want to fork each PR just to verify it. Instead of this, the reviewer can ask to include a test to verify that provided solution works. If the test was included and CI passed, the reviewer can be sure that a new feature was implemented correctly or a bug was fixed. The only important code to review is assertions accuracy in the tests but it can be verified without forking, just by looking at the source code.

**Comment by vassilevsky** [comment]
@g4s8 I have addressed your comments 🤝

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) everybody who has role `REV` is banned at [#20](https://github.com/g4s8/gitstrap/issues/20); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) everybody who has role `REV` is banned at [#20](https://github.com/g4s8/gitstrap/issues/20); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:g4s8/gitstrap#20` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#20](https://github.com/g4s8/gitstrap/issues/20) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**